src/Microsoft.Health.Dicom.Api/Controllers/CustomTagController.cs

```diff
@@ -50,5 +51,35 @@ public async Task<IActionResult> PostAsync([FromBody] IEnumerable<CustomTagEntry
             return StatusCode(
                (int)HttpStatusCode.Accepted, response);
         }
+
+        /// <summary>
+        /// Handles requests to get individual/all custom tags.
+        /// </summary>
+        /// <param name="tagPath">Path for requested custom tag. Value is null when retrieving all custom tags.</param>
+        /// <returns>
+        /// Returns Bad Request if given path can't be parsed. Returns Not Found if given path doesn't map to a stored
+        /// custom tag or if no custom tags are stored. Returns OK with a JSON body of requested tag(s) in other cases.
+        /// </returns>
+        [ProducesResponseType(typeof(JsonResult), (int)HttpStatusCode.OK)]
+        [ProducesResponseType((int)HttpStatusCode.BadRequest)]
+        [ProducesResponseType((int)HttpStatusCode.NotFound)]
+        [HttpGet]
+        [Route(KnownRoutes.CustomTagRoute)]
+        [AuditEventType(AuditEventSubType.CustomTag)]
+        public async Task<IActionResult> GetAsync(string tagPath)
```

Comment:
should the input be Id instead of tagpath?
The List api will returns all custom tag information where user can find tag id.

Reply:
What do we do in the scenario where there are multiple tag paths with the same id? I figured tag path would return the most concrete answer for the user.